proto/wg/cosmo/platform/v1/platform.proto

```diff
@@ -3002,4 +3026,6 @@ service PlatformService {
   rpc GetProposalsByFederatedGraph(GetProposalsByFederatedGraphRequest) returns (GetProposalsByFederatedGraphResponse) {}
   // GetProposalChecks returns checks for a proposal.
   rpc GetProposalChecks(GetProposalChecksRequest) returns (GetProposalChecksResponse) {}
+  // GetOperations returns the operations of a federated graph.
+  rpc GetOperations(GetOperationsRequest) returns (GetOperationsResponse) {}
```

Comment:
Why add it to the Proposal group?

Reply:
Its not in the proposal group. Its just below the proposal rpcs